### ISSUE TITLE
DATA_RATE_KIBIBYTES_PER_SECOND was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfDataRate.KIBIBYTES_PER_SECOND instead

### ISSUE THREAD

**Issue by hapklaar** [al-one/hass-xiaomi-miot#1490]
```
Logger: homeassistant.const
Source: custom_components/xiaomi_miot/switch.py:5
Integration: Xiaomi Miot Auto ([documentation](https://github.com/al-one/hass-xiaomi-miot), [issues](https://github.com/al-one/hass-xiaomi-miot/issues))
First occurred: 19:42:28 (136 occurrences)
Last logged: 19:42:28

DATA_RATE_KIBIBYTES_PER_SECOND was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfDataRate.KIBIBYTES_PER_SECOND instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
DATA_RATE_MEBIBYTES_PER_SECOND was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfDataRate.MEBIBYTES_PER_SECOND instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
DATA_RATE_GIBIBYTES_PER_SECOND was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfDataRate.GIBIBYTES_PER_SECOND instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
ENTITY_CATEGORY_CONFIG was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use EntityCategory.CONFIG instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
ENTITY_CATEGORY_DIAGNOSTIC was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use EntityCategory.DIAGNOSTIC instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
```

**Comment by codyc1515** [comment]
The following consts also need updating:

```
Logger: homeassistant.components.alarm_control_panel.const
Source: helpers/deprecation.py:205
Integration: Alarm control panel (documentation, issues)
First occurred: 08:43:39 (8 occurrences)
Last logged: 08:43:39

SUPPORT_ALARM_ARM_AWAY was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use AlarmControlPanelEntityFeature.ARM_AWAY instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
SUPPORT_ALARM_ARM_NIGHT was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use AlarmControlPanelEntityFeature.ARM_NIGHT instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
SUPPORT_ALARM_TRIGGER was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use AlarmControlPanelEntityFeature.TRIGGER instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
SUPPORT_ALARM_ARM_CUSTOM_BYPASS was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use AlarmControlPanelEntityFeature.ARM_CUSTOM_BYPASS instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
SUPPORT_ALARM_ARM_VACATION was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use AlarmControlPanelEntityFeature.ARM_VACATION instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
```

**Comment by codyc1515** [comment]
Missed in #1483

**Comment by z-master42** [comment]
Same for:
```
SUPPORT_TARGET_HUMIDITY was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.TARGET_HUMIDITY instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues

SUPPORT_FAN_MODE was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.FAN_MODE instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues

SUPPORT_PRESET_MODE was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.PRESET_MODE instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues

SUPPORT_SWING_MODE was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.SWING_MODE instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues

SUPPORT_AUX_HEAT was used from xiaomi_miot, this is a deprecated constant which will be removed in HA Core 2025.1. Use ClimateEntityFeature.AUX_HEAT instead, please create a bug report at https://github.com/al-one/hass-xiaomi-miot/issues
```

**Comment by hmmbob** [comment]
Should be fixed by #1485, which is not released yet.
